<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ImageEditDialog</class>
 <widget class="QDialog" name="ImageEditDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1028</width>
    <height>790</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>微软雅黑</family>
    <pointsize>12</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Image Editor</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QGroupBox" name="groupBox_8">
       <property name="title">
        <string>Tools</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_7">
        <item row="2" column="0">
         <widget class="QToolButton" name="imageZoomButton">
          <property name="toolTip">
           <string>Zoomin/Zoomout</string>
          </property>
          <property name="text">
           <string>...</string>
          </property>
          <property name="icon">
           <iconset resource="../resources.qrc">
            <normaloff>:/imagetool/Resources/imagetool/zoom.png</normaloff>:/imagetool/Resources/imagetool/zoom.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>48</width>
            <height>48</height>
           </size>
          </property>
          <property name="arrowType">
           <enum>Qt::NoArrow</enum>
          </property>
         </widget>
        </item>
        <item row="3" column="2">
         <widget class="QToolButton" name="imageHFllipButton">
          <property name="toolTip">
           <string>Horizontal flip</string>
          </property>
          <property name="text">
           <string>...</string>
          </property>
          <property name="icon">
           <iconset resource="../resources.qrc">
            <normaloff>:/imagetool/Resources/imagetool/hflip.png</normaloff>:/imagetool/Resources/imagetool/hflip.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>48</width>
            <height>48</height>
           </size>
          </property>
          <property name="arrowType">
           <enum>Qt::NoArrow</enum>
          </property>
         </widget>
        </item>
        <item row="3" column="3">
         <widget class="QToolButton" name="imageVFlipButton">
          <property name="toolTip">
           <string>Vertical Flip</string>
          </property>
          <property name="text">
           <string>...</string>
          </property>
          <property name="icon">
           <iconset resource="../resources.qrc">
            <normaloff>:/imagetool/Resources/imagetool/vflip.png</normaloff>:/imagetool/Resources/imagetool/vflip.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>48</width>
            <height>48</height>
           </size>
          </property>
          <property name="arrowType">
           <enum>Qt::NoArrow</enum>
          </property>
         </widget>
        </item>
        <item row="2" column="2">
         <widget class="QToolButton" name="imageRotateCCWButton">
          <property name="toolTip">
           <string>Rotate image by 90 counter clockwise</string>
          </property>
          <property name="text">
           <string>...</string>
          </property>
          <property name="icon">
           <iconset resource="../resources.qrc">
            <normaloff>:/imagetool/Resources/imagetool/rotateccw.png</normaloff>:/imagetool/Resources/imagetool/rotateccw.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>48</width>
            <height>48</height>
           </size>
          </property>
          <property name="arrowType">
           <enum>Qt::NoArrow</enum>
          </property>
         </widget>
        </item>
        <item row="2" column="3">
         <widget class="QToolButton" name="imageRotateCWButton">
          <property name="toolTip">
           <string>Rotate image by 90 clockwise</string>
          </property>
          <property name="text">
           <string>...</string>
          </property>
          <property name="icon">
           <iconset resource="../resources.qrc">
            <normaloff>:/imagetool/Resources/imagetool/rotatecw.png</normaloff>:/imagetool/Resources/imagetool/rotatecw.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>48</width>
            <height>48</height>
           </size>
          </property>
          <property name="arrowType">
           <enum>Qt::NoArrow</enum>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QToolButton" name="imageCutButton">
          <property name="toolTip">
           <string>Cut image</string>
          </property>
          <property name="text">
           <string>...</string>
          </property>
          <property name="icon">
           <iconset resource="../resources.qrc">
            <normaloff>:/imagetool/Resources/imagetool/edit_cut.png</normaloff>:/imagetool/Resources/imagetool/edit_cut.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>48</width>
            <height>48</height>
           </size>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
          <property name="arrowType">
           <enum>Qt::NoArrow</enum>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QToolButton" name="imagePanButton">
          <property name="toolTip">
           <string>Pan</string>
          </property>
          <property name="text">
           <string>...</string>
          </property>
          <property name="icon">
           <iconset resource="../resources.qrc">
            <normaloff>:/imagetool/Resources/imagetool/pan.png</normaloff>:/imagetool/Resources/imagetool/pan.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>48</width>
            <height>48</height>
           </size>
          </property>
          <property name="arrowType">
           <enum>Qt::NoArrow</enum>
          </property>
         </widget>
        </item>
        <item row="1" column="2">
         <widget class="QToolButton" name="imageUndoButton">
          <property name="toolTip">
           <string>Adjust Window width and center</string>
          </property>
          <property name="text">
           <string>...</string>
          </property>
          <property name="icon">
           <iconset resource="../resources.qrc">
            <normaloff>:/imagetool/Resources/imagetool/undo.png</normaloff>:/imagetool/Resources/imagetool/undo.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>48</width>
            <height>48</height>
           </size>
          </property>
          <property name="arrowType">
           <enum>Qt::NoArrow</enum>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QToolButton" name="imageRFlagButton">
          <property name="toolTip">
           <string>Add R flag</string>
          </property>
          <property name="text">
           <string>...</string>
          </property>
          <property name="icon">
           <iconset resource="../resources.qrc">
            <normaloff>:/imagetool/Resources/imagetool/r_flag.png</normaloff>:/imagetool/Resources/imagetool/r_flag.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>48</width>
            <height>48</height>
           </size>
          </property>
          <property name="arrowType">
           <enum>Qt::NoArrow</enum>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QToolButton" name="imageLFlagButton">
          <property name="toolTip">
           <string>Add L flag</string>
          </property>
          <property name="text">
           <string>...</string>
          </property>
          <property name="icon">
           <iconset resource="../resources.qrc">
            <normaloff>:/imagetool/Resources/imagetool/l_flag.png</normaloff>:/imagetool/Resources/imagetool/l_flag.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>48</width>
            <height>48</height>
           </size>
          </property>
          <property name="arrowType">
           <enum>Qt::NoArrow</enum>
          </property>
         </widget>
        </item>
        <item row="0" column="3">
         <widget class="QToolButton" name="imageFlagFontButton">
          <property name="toolTip">
           <string>Flag font</string>
          </property>
          <property name="text">
           <string>...</string>
          </property>
          <property name="icon">
           <iconset resource="../resources.qrc">
            <normaloff>:/imagetool/Resources/imagetool/textfont.png</normaloff>:/imagetool/Resources/imagetool/textfont.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>48</width>
            <height>48</height>
           </size>
          </property>
          <property name="arrowType">
           <enum>Qt::NoArrow</enum>
          </property>
         </widget>
        </item>
        <item row="1" column="3">
         <widget class="QToolButton" name="imageRedoButton">
          <property name="toolTip">
           <string>Line</string>
          </property>
          <property name="text">
           <string>...</string>
          </property>
          <property name="icon">
           <iconset resource="../resources.qrc">
            <normaloff>:/imagetool/Resources/imagetool/redo.png</normaloff>:/imagetool/Resources/imagetool/redo.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>48</width>
            <height>48</height>
           </size>
          </property>
          <property name="arrowType">
           <enum>Qt::NoArrow</enum>
          </property>
         </widget>
        </item>
        <item row="7" column="2">
         <widget class="QToolButton" name="imageArrowButton">
          <property name="toolTip">
           <string>Arrow</string>
          </property>
          <property name="text">
           <string>...</string>
          </property>
          <property name="icon">
           <iconset resource="../resources.qrc">
            <normaloff>:/imagetool/Resources/imagetool/arrow.png</normaloff>:/imagetool/Resources/imagetool/arrow.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>48</width>
            <height>48</height>
           </size>
          </property>
          <property name="arrowType">
           <enum>Qt::NoArrow</enum>
          </property>
         </widget>
        </item>
        <item row="7" column="0">
         <widget class="QToolButton" name="imageAnnotationButton">
          <property name="toolTip">
           <string>Text</string>
          </property>
          <property name="text">
           <string>...</string>
          </property>
          <property name="icon">
           <iconset resource="../resources.qrc">
            <normaloff>:/imagetool/Resources/imagetool/Text.png</normaloff>:/imagetool/Resources/imagetool/Text.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>48</width>
            <height>48</height>
           </size>
          </property>
          <property name="arrowType">
           <enum>Qt::NoArrow</enum>
          </property>
         </widget>
        </item>
        <item row="6" column="3">
         <widget class="QToolButton" name="imageRectangleButton">
          <property name="toolTip">
           <string>Rectangle</string>
          </property>
          <property name="text">
           <string>...</string>
          </property>
          <property name="icon">
           <iconset resource="../resources.qrc">
            <normaloff>:/imagetool/Resources/imagetool/rectangle.png</normaloff>:/imagetool/Resources/imagetool/rectangle.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>48</width>
            <height>48</height>
           </size>
          </property>
          <property name="arrowType">
           <enum>Qt::NoArrow</enum>
          </property>
         </widget>
        </item>
        <item row="6" column="2">
         <widget class="QToolButton" name="imageEllipseButton">
          <property name="toolTip">
           <string>Ecllipse</string>
          </property>
          <property name="text">
           <string>...</string>
          </property>
          <property name="icon">
           <iconset resource="../resources.qrc">
            <normaloff>:/imagetool/Resources/imagetool/ecllipse.png</normaloff>:/imagetool/Resources/imagetool/ecllipse.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>48</width>
            <height>48</height>
           </size>
          </property>
          <property name="arrowType">
           <enum>Qt::NoArrow</enum>
          </property>
         </widget>
        </item>
        <item row="6" column="0">
         <widget class="QToolButton" name="imageCobbAngleButton">
          <property name="toolTip">
           <string>Cobbangle</string>
          </property>
          <property name="text">
           <string>...</string>
          </property>
          <property name="icon">
           <iconset resource="../resources.qrc">
            <normaloff>:/imagetool/Resources/imagetool/cobbangle.png</normaloff>:/imagetool/Resources/imagetool/cobbangle.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>48</width>
            <height>48</height>
           </size>
          </property>
          <property name="arrowType">
           <enum>Qt::NoArrow</enum>
          </property>
         </widget>
        </item>
        <item row="5" column="3">
         <widget class="QToolButton" name="imageAngleButton">
          <property name="toolTip">
           <string>Angle</string>
          </property>
          <property name="text">
           <string>...</string>
          </property>
          <property name="icon">
           <iconset resource="../resources.qrc">
            <normaloff>:/imagetool/Resources/imagetool/angle.png</normaloff>:/imagetool/Resources/imagetool/angle.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>48</width>
            <height>48</height>
           </size>
          </property>
          <property name="arrowType">
           <enum>Qt::NoArrow</enum>
          </property>
         </widget>
        </item>
        <item row="5" column="2">
         <widget class="QToolButton" name="imageRulerButton">
          <property name="toolTip">
           <string>Ruler</string>
          </property>
          <property name="text">
           <string>...</string>
          </property>
          <property name="icon">
           <iconset resource="../resources.qrc">
            <normaloff>:/imagetool/Resources/imagetool/length.png</normaloff>:/imagetool/Resources/imagetool/length.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>48</width>
            <height>48</height>
           </size>
          </property>
          <property name="arrowType">
           <enum>Qt::NoArrow</enum>
          </property>
         </widget>
        </item>
        <item row="5" column="0">
         <widget class="QToolButton" name="imageNegativeButton">
          <property name="toolTip">
           <string>Negative image</string>
          </property>
          <property name="text">
           <string>...</string>
          </property>
          <property name="icon">
           <iconset resource="../resources.qrc">
            <normaloff>:/imagetool/Resources/imagetool/negative.png</normaloff>:/imagetool/Resources/imagetool/negative.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>48</width>
            <height>48</height>
           </size>
          </property>
          <property name="arrowType">
           <enum>Qt::NoArrow</enum>
          </property>
         </widget>
        </item>
        <item row="7" column="3">
         <widget class="QToolButton" name="imageItemDeleteButton">
          <property name="toolTip">
           <string>Delete</string>
          </property>
          <property name="text">
           <string>...</string>
          </property>
          <property name="icon">
           <iconset resource="../resources.qrc">
            <normaloff>:/imagetool/Resources/imagetool/delete.png</normaloff>:/imagetool/Resources/imagetool/delete.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>48</width>
            <height>48</height>
           </size>
          </property>
          <property name="arrowType">
           <enum>Qt::NoArrow</enum>
          </property>
         </widget>
        </item>
        <item row="8" column="0">
         <widget class="QToolButton" name="imageResetButton">
          <property name="toolTip">
           <string>Reset</string>
          </property>
          <property name="text">
           <string>...</string>
          </property>
          <property name="icon">
           <iconset resource="../resources.qrc">
            <normaloff>:/imagetool/Resources/imagetool/reset.png</normaloff>:/imagetool/Resources/imagetool/reset.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>48</width>
            <height>48</height>
           </size>
          </property>
          <property name="arrowType">
           <enum>Qt::NoArrow</enum>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QCommandLinkButton" name="imageOkButton">
       <property name="styleSheet">
        <string notr="true">font: 12pt &quot;微软雅黑&quot;;</string>
       </property>
       <property name="text">
        <string>Accept Image</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCommandLinkButton" name="imageCancelButton">
       <property name="styleSheet">
        <string notr="true">font: 12pt &quot;微软雅黑&quot;;</string>
       </property>
       <property name="text">
        <string>Reject Image</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../resources.qrc"/>
 </resources>
 <connections/>
</ui>
